spec/controllers/search_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require "rails_helper"
+
```

Comment:
Do these tests scan the responses for certain pieces of information, or just look at the HTTP response code?